apps/web/src/scenes/Modals/PaginatedList/PaginatedUsersList.tsx

```diff
@@ -0,0 +1,117 @@
+import { Route } from 'nextjs-routes';
+import { useCallback, useMemo } from 'react';
+import { graphql, usePaginationFragment } from 'react-relay';
+import { AutoSizer, InfiniteLoader, List, ListRowRenderer } from 'react-virtualized';
+import styled from 'styled-components';
+
+import { VStack } from '~/components/core/Spacer/Stack';
+import { PaginatedUsersListFragment$key } from '~/generated/PaginatedUsersListFragment.graphql';
+import { useIsMobileWindowWidth } from '~/hooks/useWindowSize';
+import unescape from '~/utils/unescape';
+
+import PaginatedListRow from './PaginatedListRow';
+
+type Props = {
+  queryRef: PaginatedUsersListFragment$key;
+};
+
+export const FOLLOWERS_PER_PAGE = 20;
+
+export default function PaginatedUsersList({ queryRef }: Props) {
+  const { data, loadNext, hasNext } = usePaginationFragment(
+    graphql`
+      fragment PaginatedUsersListFragment on GalleryUser
+      @refetchable(queryName: "RefetchablePaginatedUsersListFragment") {
+        sharedFollowers(first: $sharedFollowersFirst, after: $sharedFollowersAfter)
+          @connection(key: "PaginatedUsersListFragment_sharedFollowers") {
+          edges {
+            node {
+              __typename
+
+              username
+              bio
+            }
+          }
+        }
+      }
+    `,
+    queryRef
+  );
```

Comment:
This file is in the `src/scenes/Modals` which makes me think you'd want to re-use this in a few places. However, we're fetching "sharedFollowers", which seems awfully specific.

We already have `FollowListUsers`, maybe that could be adapted. I feel like we could do some consolidation there. 

Takeaway here is, we should either figure out how to consolidate these, or just rename this component since it's not generic.

Reply:
moved these files out of `/Modals` and renamed named them to be specific to the Shared Info piece of User Profiles